PWGGA/GammaConvBase/AliConversionMesonCuts.cxx

```diff
@@ -126,6 +126,7 @@ AliConversionMesonCuts::AliConversionMesonCuts(const char *name,const char *titl
   fMesonKind(0),
   fIsMergedClusterCut(0),
   fUsePtDepSelectionWindow(kFALSE),
+  fUseGammaSelection(kFALSE),
```

Comment:
Why did you add this variable? It is not being used...

Reply:
It should be used in SetSelectionWindowCutPtDep for multiple cases

Reply:
Maybe i'm overlooking something, but it is only set to kTRUE there but the variable is not used for anything.

Reply:
there is also the function 
```C++
Bool_t   UseGammaSelection() const{return fUseGammaSelection;}
```
in AliConversionMesonCuts.cxx
which is called in the OmegaToPiZeroGamma Task to check if the flag is set or not.

Reply:
I figured it out :)